src/Util/FilterUtil.ts

```diff
@@ -218,6 +218,15 @@ class FilterUtil {
       counts: [],
       duplicates: []
     };
+
+    // Add id to feature if missing
+    data.exampleFeatures.features = data.exampleFeatures.features.map((feature, idx) => {
+      if (!feature.id) {
+        feature.id = idx;
```

Comment:
[Lodash's](https://lodash.com/docs/4.17.11#uniqueId) `uniqueId` function could help to generate unique ids (e.g. with feature type as prefix), what do you think?

Reply:
That was my first thought, but i think this is should be faster and fit our needs. I think there will be no FeatureCollection with "mixed content" where some features have ids and some not.